chatmeleon-server/src/controllers/user-relationship-controller.ts

```diff
@@ -0,0 +1,125 @@
+import prisma from "../libs/prismadb";
+import { Request, Response } from "express";
+
+const createFriendRequestRelationship = async (req: Request, res: Response) => {
+  const { relatingUserId, relatedUserId } = req.body;
```

Comment:
Missing input validation

Reply:
mot tinh bro lol

Reply:
Yeah it makes more sense to write a friend API first but i was a bit lazy lol. Can this be an exception UWU, only this time yea? plss next time promise do properly yes

Reply:
> Yeah it makes more sense to write a friend API first but i was a bit lazy lol. Can this be an exception UWU, only this time yea? plss next time promise do properly yes

Yeah sure lol

Reply:
> Also, there will be a huge change in our API implementations since now we getting the requesting user's ID through the auth header so I just don't want to take the liberty of changing the API without consultants maybe this Thursday we can resolve this?

I think it really depends on which PR gets merged first. I guess there really should be someone taking charge of managing these decisions in the project. lol

Reply:
Okay then this thursday we will revisit our team work flow and re-organize Jira again since its been inactive for a long time